code-studio/src/js/components/GridEditor.jsx

```diff
@@ -83,9 +83,19 @@ var GridEditor = React.createClass({
     });
   },
 
-  handleCellChange: function (newSerializedCell) {
-    var row = this.state.selectedRow;
-    var col = this.state.selectedCol;
+  /**
+   * Helper method used to update chunks of the grid. Excepts a row and
```

Comment:
Haha, I think I was actually trying to decide between "accepts" and "expects" and instead wrote neither.